twittersearch/api_utils.py

```diff
@@ -251,6 +263,4 @@ def load_credentials(filename=None, account_type=None):
         search_args = {"username": search_creds["username"],
                        "password": search_creds["password"],
                        "endpoint": search_creds["endpoint"]}
-    count_args = {**search_args,
-                  **{"endpoint": change_to_count_endpoint(search_args["endpoint"])}}
-    return search_args, count_args
+    return search_args
```

Comment:
Good simplification of the return type here. 

Based on this new version, I think the one additional thing I'd like to see in this method is user-friendly exception handling for the KeyErrors that will come from a misconfigured yaml file. The simplest version might be a try/except (just for `KeyError`) around the two conditional statements with a helpful error message along the lines of "ensure that your credentials file has the proper keys and values for the endpoint you're trying to use. see the readme for examples."